scripts/get-strata-and-uuids-from-ecqm-zip.js

```diff
@@ -62,12 +62,14 @@ return strata name, description, and uuids like so
   ...
 */
 function extractStrata(measure) {
-  const details = measure.measure.measureDetails[0];
   // our version of 'strata' are described as 'numerators'
   // parse out strata descriptions from numerator text
-  const description = details.numeratorDescription[0];
   // descriptions are like "Numerator 1: Patients who initiated treatment within 14 days of the diagnosis\nNumerator 2: Patients who initiated treatment and who had two or more additional services with an AOD diagnosis within 30 days of the initiation visit"
-  let strataDescriptions = _.compact(description.split('\n'))
+  const description = measure.subjectOf
+    .find(item => item.measureAttribute[0].code[0].$.code === 'NUMER')
+    .measureAttribute[0].value[0].$.value;
+
+  let strataDescriptions = _.compact(description.split(/\n|\r|&#xA;/))
     // if multiple strata, they're enumerated as 'Numerator x: '
```

Comment:
I switched from parsing CMS1v2_Simple.xml files to CMS1v2.xml files, and the latter uses `&#xA;` (html entity for line feed character) to delimit some but not all strata descriptions :\